SlowCheetah/SlowCheetah.csproj

```diff
@@ -86,7 +86,9 @@
     <None Include="Build\readme.txt">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
-    <None Include="packages.config" />
+    <None Include="packages.config">
```

Comment:
Consider the newer world of project.json or the much better and latest world of PackageReference instead of packages.config. It requires VS 2017, but hey, we shipped it. :)